### ISSUE TITLE
Build Docs Site CI has suddenly become very slow

### ISSUE THREAD

**Issue by vorburger** [enola-dev/enola#520]
Up to yesterday, the _Build Docs Site_ step on CI consistently ran in ~3'.

I noticed during the (final) build of #519 that this (massively) increased to ~X' now.

The most likely explanation may be that the Runner VM is running out of memory?

But it's not clear to me which change on #519 could have possibly caused this...

... it would need some exploration with Reverts on PRs to debug further (starting with reverting all those 6 commits).

Actually the addition of the JVM to be installed by ASDF is the most likely culprit?

Could `openjdk-21` (ASDF) really be that much slower than the Temurin one previously installed by `.github/workflows/ci.yaml`?!

BTW This needs further clean-up - we should now be able to remove the Java installation by GitHub Action. TBD: Caching?

Or maybe it's something else totally unrelated.

Or GitHub is just throttling us today... 🤣 

TL;DR I'll merge #519 and check this later.

**Comment by vorburger** [comment]
[This `main` build](https://github.com/enola-dev/enola/actions/runs/7982679331/job/21796613327) also spent 13m 48s (!) in _Build Docs Site_ so clearly not some one off "bad VM" kind of fluke... hm.

**Comment by vorburger** [comment]
> Could openjdk-21 (ASDF) really be that much slower than the Temurin one previously installed by .github/workflows/ci.yaml?!

That's not it, in #522 with this ^^^ it's still 14m 18s... hm.
